### DISCUSSION TITLE
Use `clx` to browse non-hacker-news websites? :D

### DISCUSSION THREAD

**Discussion by hacker-DOM** [bensadeh/circumflex#31] [Ideas]
The reader functionality is really cool! Would it be possible to use it to browse a url? (clx --browse www.nytimes.com/bla) Also, do you know of any tools that provide a similar function?

**Comment by bensadeh** [comment]
Glad you like the reader functionality! What you're describing is not currently possible. It really depends on the API the site is offering, so if the API is giving us a list of stories, then it shouldn't be difficult to adapt it to work in `circumflex` if it is something you'd like to try.

**Reply by hacker-DOM** [reply]
Thanks for the answer @bensadeh. I actually meant just the feature of turning a web page into cli-enabled markdown! 

I looked briefly into the codebase and as a Go noob, I don't dare to try to create it :-) However, I think it could be useful.

We already have read and view, so maybe `clx browse [url]`? Or even `clx read -u [url]`.

**Reply by bensadeh** [reply]
Hey @hacker-DOM ,

I just added support for this in `3.3`! Any URL that supports Reader Mode can be viewed in the terminal like so:
```terminal
clx url [URL]
```

I also renamed some of the existing commands as they started to become a bit confusing. Hope you enjoy the feature 😎
